libraries/bxdf/lama/lama_conductor.mtlx

```diff
@@ -1,34 +1,20 @@
 <?xml version="1.0"?>
 <materialx version="1.38" colorspace="acescg">
   <nodedef name="ND_lama_conductor" node="LamaConductor" nodegroup="pbr" doc="Lama conductor" version="1.0" isdefaultversion="true">
-    <input name="tint" type="color3" value="1, 1, 1" uiname="Tint" uifolder="Main"
-           doc="Overall color multiplier. It should be used with parcimony, as a non-white value breaks physicality. The prefered way to define the color of a conductor is through the Fresnel attributes right below." />
```

Comment:
@kwokcb I like the idea of this standardization pass on the data libraries and examples, and I have two recommendations for improvements:

1. In some situations, including the highlighted example text, we're using newlines before `doc` strings to enhance readability.  In these cases, I would recommend leaving the original formatting of these sections untouched, despite our not having a way to retain the formatting automatically.  By analogy, we take a similar approach when applying Clang Format to our C++ codebase, where certain formatting structures designed for readability (e.g. displaying 4x4 matrices as 4 rows of 4 numbers) are overwritten by the Clang Format process, and we need to restore them manually after applying our formatting step.
2. Since the new `mxformat.py` is substantially the same as our existing `mxupdate.py`, I would recommend merging these into one helper script.  I like your suggestion of naming this script `mxformat.py`, and perhaps it should simply have a command-line option that selects whether version upgrades are also applied to the selected set of MaterialX documents.

Reply:
This sounds good. I'll make the changes. 
This is deprecate `mxupdate` so users will need to be notified of this.